ncm-network/src/main/perl/network.pm

```diff
@@ -1544,6 +1544,47 @@ sub legacy_keeps_state
     };
 }
 
+
+# check if file change was due to removal of explicit broadcast in favour of computed default
+#   (broadcast configuration has a default via "ipcalc --broadcast ipaddr mask")
+sub default_broadcast_keeps_state
+{
+    my ($self, $file_name, $name, $data, $exifiles, $alias) = @_;
+
+    # compute broadcast via ipcalc, remove BROADCAST=
+    my $proc = CAF::Process->new(
+        ["ipcalc", "--broadcast", $data->{ip}, $data->{netmask}],
```

Comment:
I presume it is reasonable to assume that `ipcalc` is installed?

Reply:
it is the code run by initscripts, so i'd say yes

Reply:
Ok, it's provided by the initscripts package in RH-land.
It is not installed by default on debian (if we care i) - it is however available in the ipcalc package.